tslint.json

```diff
@@ -0,0 +1,63 @@
+{
+  "rules": {
+    "class-name": true,
+    "comment-format": [true, "check-lowercase", "check-space"],
+    "curly": true,
+    "eofline": true,
+    "forin": true,
+    "indent": [true, "spaces"],
+    "label-position": true,
+    "label-undefined": true,
+    "max-line-length": [true, 140],
+    "member-ordering": [true,
+      "public-before-private",
+      "static-before-instance",
+      "variables-before-functions"
+    ],
+    "no-arg": true,
+    "no-bitwise": true,
+    "no-console": [true,
+      "debug",
+      "info",
+      "time",
+      "timeEnd",
+      "trace"
+    ],
+    "no-construct": true,
+    "no-debugger": true,
```

Comment:
Same here, this is dev environment so access to `debugger` makes sense.
May be we can create two separate `tslint` config files when we create a production build (once ng2 is officially released).